emission/tests/netTests/TestWebserver.py

```diff
@@ -23,6 +24,40 @@
 
 class TestWebserver(unittest.TestCase):
     def setUp(self):
+        # Backwards_Compatibility method to read from config files 
+        self.webserver_conf_path = "conf/net/api/webserver.conf"
```

Comment:
when will we ever have a `webserver.conf` file that we need to back up in a test environment?

Reply:
> when will we ever have a webserver.conf file that we need to back up in a test environment?

After clarification regarding the testing environments, I do not think we'll have this situation and won't need changes in `TestWebserver.py`

> Note also that we currently support testing in three environments:
In a linux CI/CD environment, using docker compose
In a linux CI/CD environment, running from a local install
In a Mac environment, running from a local install

The keywords here are "test environment". 
I had assumed that we might have conf files present in test environment in case someone wanted to check whether values were being read correctly from the config file. 
If the test was run in the presence of webserver.conf file then it was failing and hence I thought it was something to fix.

But if it is true that we will not have any conf (non-sample) files in testing environments, then the changes related to `TestWebserver.py` are not required.

Reply:
I do not think we need to consider the use case of an arbitrary config file in automated tests. The whole point of the automated tests is that they run in a known, reproducible environment and generate known, reproducible results. This is particularly true for backwards compatible code, which will be removed soon.

Having said all that, more tests are not bad. As I said "This is not incorrect, but I am not sure why it is required.". If you can make the case for having this test (what is it testing that the existing tests are not?), we can include it.

Reply:
We can skip the changes in the test since I wasn't adding any new tests. I had just added code that ensured we read from the config file (non-sample) if present. But it's clear to me now that this isn't necessary in a known testing environment.